.github/workflows/buildAndTestRyzenAI.yml

```diff
@@ -37,12 +37,13 @@ concurrency:
 env:
   DEBIAN_FRONTEND: noninteractive
   XILINXD_LICENSE_FILE: /opt/xilinx/Xilinx.lic
+  VITIS: /opt/xilinx/Vitis/2023.2
 
 jobs:
   build-repo:
     name: Build and Test with AIE tools on Ryzen AI
 
-    runs-on: amd7940hs
+    runs-on: amd8700g
```

Comment:
Should we have `runs-on: [amd7940hs, amd8700g]` here?

Reply:
I think so

Reply:
Sorry that was a swing and a miss by me. That syntax means match ALL the labels.  We could set it to `self-hosted`, but maybe we need a different label `npu` or `ryzenai` or whatever to encompass both

Reply:
I created label `ryzenai` for both runner types

Reply:
My sense is that unless you're really running the same OS image (e.g. a container), that having multiple runners on the same name is a bad idea.  Github just doesn't provide enough management/debug of runners to understand which particular instance of a runner one is running on.